activiti-services/activiti-services-connectors/src/main/java/org/activiti/services/connectors/channel/ServiceTaskResultHandler.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.connectors.channel;
+
+import org.activiti.engine.RuntimeService;
+import org.activiti.services.connectors.model.ServiceTaskResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.annotation.StreamListener;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableBinding(ServiceTaskChannels.class)
+public class ServiceTaskResultHandler {
+
+    private final RuntimeService runtimeService;
+
+    @Autowired
+    public ServiceTaskResultHandler(RuntimeService runtimeService) {
+        this.runtimeService = runtimeService;
+    }
+
+    @StreamListener(ServiceTaskChannels.SERVICE_TASK_RESULT_CONSUMER)
+    public synchronized void receive(ServiceTaskResult serviceTaskResult) {
```

Comment:
@erdemedeiros same here.. the types should be modelled as events and I think that IntegrationEvent and IntegrationResult makes more sense as names.. Because this should not be related with the BPMN Service Task Concept. We might be able to generate integration events from other places.. not only ServiceTasks

Reply:
ok